test/mini-graphql-http-client.js

```diff
@@ -93,4 +104,93 @@ describe("MiniGraphqlHttpClient tests", () => {
             expect(cache2.map.get("2421565178")).to.have.deep.property("json", { bla: 1 });
         });
     });
+
+    describe("#retry", () => {
+        it("should try making 6 retries on 5xx responses but stop on 5-th try because of good response", async function () {
+            let wasError = false;
+            let numberOfRetries = 6;
+            const cache = {};
+            const fakeFetch = createFakeFetch([500, 500, 500, 500, 200, 500]);
```

Comment:
I believe variables `numberOfRetries` and `fakeFetch` are  unnecessary. This is more readable as to my taste:
```
            const client = MiniGraphqlHttpClient({
                uri: "https://a.aa",
                fetch: createFakeFetch([500, 500, 500, 500, 200]),
                retry: 6,
            });
```